.github/workflows/kind-cluster-image-policy.yaml

```diff
@@ -53,7 +55,7 @@ jobs:
 
     env:
       KO_DOCKER_REPO: "registry.local:5000/policy-controller"
-      SCAFFOLDING_RELEASE_VERSION: "v0.6.5"
+      SCAFFOLDING_RELEASE_VERSION: "v0.6.17"
```

Comment:
cut a new release for scaffolding: https://github.com/sigstore/scaffolding/releases/tag/v0.7.1

but that we will need to drop 1.25/1.26 from the matrix